merp_reservation_strategy/views/res_config.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_stock_config_settings" model="ir.ui.view">
+        <field name="name">Ventor/mERP Picking Wave - Stock Settings</field>
+        <field name="model">res.config.settings</field>
+        <field name="inherit_id" ref="base.res_config_settings_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[@data-key='merp_base']//field[@name='outgoing_routing_strategy']/parent::div/parent::div" position="before" >
+                    <div class="col-xs-12 col-md-6 o_setting_box">
+                        <div class="o_setting_left_pane">
+                            <field name="custom_reservation_strategy"
+                                attrs="{'invisible': [('module_merp_outgoing_routing','=',False)]}"/>
+                        </div>
+                        <div class="o_setting_right_pane">
+                            <label for="custom_reservation_strategy"/>
+                            <div class="text-muted">Custom Stock Reservation Strategy</div>
+                        </div>
+                    </div>
+                    <div class="col-xs-12 col-md-6 o_setting_box">
+                    </div>
+            </xpath>
+        </field>
+    </record>
```

Comment:
<img width="1282" alt="Screen Shot 2020-04-24 at 16 17 52" src="https://user-images.githubusercontent.com/10652233/80217280-1bc6c480-8648-11ea-80e8-7dab3bb3ff9f.png">

This checkbox looks like it should influence other settings. Maybe we should move it below other options? Because now seems that this checkbox and below radiobuttons are related to each other